Utils/Dataflow/pyDKB/dataflow/stage/ProcessorStage.py

```diff
@@ -156,8 +156,7 @@ def defaultArguments(self):
                           nargs='?',
                           help=u'Directory for output files '
                                 '(local or HDFS). ',
-                          default='',
-                          const='output/',
+                          default='out',
                           metavar='DIR',
```

Comment:
I've found a small typo in this commit -- '...ones by defau(l)t'. :)

Reply:
Right, will fix. Thank you!

However, I have a comment too ;) it is not obvious, which commit you`re talking about, when looking at the "conversation" tab. Here it can be found easily by context (yet still should be found), but in some cases it might not work and will make one to scan through commits or check "blame" version of the file to see where this change belongs, or something.

And your comment is not about the change in the code, but cites some changes (as you used the option of leaving a comment to the code).. it makes one at the first glance to worry about the code and distructs from the real intent of the message :)